lib/mayaUsdAPI/python/wrapProxyShapeNotice.cpp

```diff
@@ -0,0 +1,122 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <mayaUsdAPI/proxyShapeNotice.h>
+
+#include <pxr/base/tf/instantiateSingleton.h>
+#include <pxr/base/tf/instantiateType.h>
+
+#include <boost/python/def.hpp>
+
+PXR_NAMESPACE_OPEN_SCOPE
```

Comment:
I know this is an implementation-only file, but it feels awkward that this is in the Pixar namespace.